conda.recipe/meta.yaml

```diff
@@ -8,6 +9,11 @@ source:
   path: ..
 
 build:
+  # Use a build number difference to ensure that the plugin
+  # variant is slightly preferred by conda's solver.
+  number: {{ number + 100 }}  # [variant=="plugin"]
```

Comment:
track_features?

Reply:
@danpetry absolutely not :-) track_features should be burned with fire. This approach was recommended by @jamiergp

Reply:
ok, thanks for the info. I'm using track_features on something I'm working on currently. Why is this preferable to using track_features, for my understanding?

Reply:
(not very many packages in defaults)

https://metayaml-conda.fly.dev/metayaml?sql=select+*+from+repodata_packages+where+index_json+like+%27%25%22track_features%22%25%27

https://metayaml-conda.fly.dev/metayaml?sql=select+*+from+repodata_packages+where+index_json+like+%27%25%22features%22%25%27